docs/rest_api.md

```diff
@@ -416,6 +416,29 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`PUT /node/<node>/boot_device`
+
+Sets the node's next boot device persistently
+
+The request body consists of JSON with a `bootdev` argument:
+
+Request body:
+    {
+
+    'bootdev': <boot device>
+
+    }
```

Comment:
* Use double quotes; single quotes aren't valid JSON.
* Indent the fields relative to the braces
* I don't think you need the extra blank lines between the braces and the field.